tsconfig.json

```diff
@@ -0,0 +1,16 @@
+{
+  "compilerOptions": {
+    //"rootDir": "app",
+    "outDir": "./dist/", // path to output directory
+    "sourceMap": true, // allow sourcemap support
+    //"strictNullChecks": true, // enable strict null checks as a best practice
+    "noImplicitAny": true, // disallow implicit any type
+    "module": "es6", // specifiy module code generation
+    "target": "es6", // specify ECMAScript target version
+    "jsx": "react", // use typescript to transpile jsx to js
+    //"allowJs": true // allow a partial TypeScript and JavaScript codebase
+    //"allowSyntheticDefaultImports": true,
```

Comment:
If we don't need a particular config, it should be removed, not commented out.

Reply:
I was going to ask you about this -- I'm not sure what the best config was so I left the options there

Reply:
We can always google this stuff if we need to reference it :)

Reply:
OK